### DISCUSSION TITLE
Clusterの表情エモート用の表情設定枠を増やしていただきたいです。

### DISCUSSION THREAD

**Discussion by hiyakake** [esperecyan/VRMConverterForVRChat#76] [Ideas]
VRChat to VRMをする際のメニューに、Clusterの表情エモート用の表情設定枠を増やしていただきたいです。

**Comment by esperecyan** [comment]
特定のサービス特有の拡張設定は実装しないか優先度が低くなりますが、「Clusterの表情エモート用の表情設定枠」というのは、具体的にはどういったものですか？　ドキュメントなどはあるでしょうか。
